spec/AppBundle/UseCase/RegisterUserUseCaseSpec.php

```diff
@@ -3,26 +3,26 @@
 namespace spec\AppBundle\UseCase;
 
 use AppBundle\Entity\User;
-use AppBundle\UseCase\Command\RegisterUserCommand;
-use AppBundle\UseCase\RegisterUserUseCase;
-use Core\Repository\UserRepositoryInterface;
+use AppBundle\UseCase\RegisterUserCommand;
+use AppBundle\UseCase\RegisterUser;
+use Core\Repository\UserRepository;
```

Comment:
ide tells us that this is interface on every step, so no need to add such suffix

Reply:
It's about the concept. I prefer when every interface has it's suffix.

Reply:
ok, I will rollback this